Simbody/include/simbody/internal/Force_Custom.h

```diff
@@ -247,6 +247,13 @@ class SimTK_SIMBODY_EXPORT Force::Custom::Implementation {
     virtual bool dependsOnlyOnPositions() const {
         return false;
     }
+    /**
+     * Called during the realizeSubsystemDynamicsImpl method in GenralForceSubsystemRep, used to determine whether a custom force
+     * should be parallelized. By default, this method returns false and can be overwritten by the force subclass.
+     */
+    virtual bool isParallelByDefault() const {
```

Comment:
It might be a little more complicated than that, since forces that are expensive but not threadsafe should also not be parallelized.